packages/legacy/core/App/container-api.ts

```diff
@@ -149,6 +150,7 @@ export type TokenMapping = {
   [TOKENS.SCREEN_SCAN]: React.FC
   [TOKENS.SCREEN_USE_BIOMETRY]: React.FC
   [TOKENS.SCREEN_ONBOARDING]: typeof Onboarding
+  [TOKENS.SCREEN_CONTACT_DETAILS]: { showConnectedTime: boolean; editContact: boolean }
```

Comment:
Similar to the other PR, could these be put in the `TOKENS.CONFIG` since they are boolean config options and not an injectable screen?